lib/src/console/command/set_command.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:cider/src/console/command/application_command.dart';
+import 'package:cider/src/console/console.dart';
+import 'package:cider/src/console/exit_code.dart';
+import 'package:pub_semver/pub_semver.dart';
+
+class SetCommand extends ApplicationCommand {
+  SetCommand(this._console);
+
+  final Console _console;
+
+  @override
+  final description = 'Setting given version explicitly';
+
+  @override
+  final name = 'set';
+
+  @override
+  final aliases = ['switch', 'change'];
+
+  @override
+  int run() {
+    try {
+      if (argResults.rest.isEmpty) {
+        throw UnsupportedError('Version must be specified');
+      } else if (argResults.rest.length > 1) {
+        throw UnsupportedError('Too many arguments passed');
+      } else {
+        Version version;
+        try {
+          version = Version.parse(argResults.rest.single);
+        } on dynamic catch (_) {
+          throw UnsupportedError('Can\'t parse version');
+        }
+        _console.log(createApp().set(version));
+      }
+    } on UnsupportedError catch (e) {
```

Comment:
Error management logic is currently [here](https://github.com/f3ath/cider/blob/master/bin/cider.dart#L12). I'd prefer to keep it consolidated.

Reply:
I have styled the error output to the console C:

Also `runZoned` in main much handier and prettier for catching global errors, in my opinion)